iiif_prezi3/helpers/make_collection.py

```diff
@@ -0,0 +1,18 @@
+from ..skeleton import Collection
+from ..loader import monkeypatch_schema
+
+
+class MakeCollection:
+    """
+    Creates a new collection, adds it to the calling Collection `items`
+    and returns the newly created object. Accepts keyword arguments to 
+    customize the resulting instance.
+    """
+
+    def make_collection(self, **kwargs):
+        child_collection = Collection(**kwargs)
```

Comment:
Maybe we could add the docstring right below the method instead of documenting the class. What do you think?

Reply:
oops, my bad. It was supposed to be inside the method indeed, I'll fix it.